packages/studio-plugin/src/utils/TypeGuards.ts

```diff
@@ -122,6 +123,8 @@ export default class TypeGuards {
               : !metadata.required
           )
         );
+      case PropValueType.TailwindClass:
+        return typeof value === "string" || value === undefined;
```

Comment:
It's not possible right now, studio-plugin isn't able to read the tailwind config. I tried to but we're not able to dynamic import .ts files due vite running in cjs. I had a hack previously by using `vm.runInNewContext` which is essentially a safer eval but I don't want to do that.

From a higher level I think additional validation here would be messy, since it wouldn't be in sync with what typescript expects (which is any string) so a developer might add other tailwind classes outside of their custom theme, and we wouldn't want to error out in those cases.